### PR TITLE
Ability to delete reading history

### LINKED ISSUE #1799: [Feature Request] Option to bulk clear recently read
**App version:**
0.8.0
**Issue/Request:**
An option to bulk clear recently read manga in the Recently read section

**Other details:**

### PR DESCRIPTION
**Pull request by ghostbear**
Closes #1799

**What it does**
It adds the ability to delete all reading history with one button

### CODE REVIEW

**Comment by joseph619** [comment]
Could you also add an undo option for this? If some user does this by mistake/curiosity, they would lose all history if they didn't have auto backup enabled.

**Comment by ghostbear** [comment]
> Could you also add an undo option for this? If some user does this by mistake/curiosity, they would lose all history if they didn't have auto backup enabled.

There is already a warning which warns users that the history data will be deleted before they confirm the deletion. And from how the restore backup works they could use it to restore history

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -712,5 +712,9 @@
     <string name="tapping_inverted_horizontal">Horizontal</string>
     <string name="tapping_inverted_vertical">Vertical</string>
     <string name="tapping_inverted_both">Both</string>
+    <string name="clear_history_completed">History deleted</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:715]: Can you group these with the other Advanced section strings, e.g. `battery_optimization_setting_activity_not_found`?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -712,5 +712,9 @@
...
+    <string name="clear_history_confirmation">Are you sure? All history will be lost</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:716]: Nitpick: add a period at the end.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -712,5 +712,9 @@
...
+    <string name="pref_clear_history">Clear history</string>
+    <string name="pref_clear_history_summary">Delete history for all manga</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:718]: "reading history"?

**Review by arkon — CHANGES_REQUESTED** [review]

**Pull request merged**